frontend/src/views/admin/matching/actions.ts

```diff
@@ -245,11 +245,22 @@ export const matchesSelector = createSelector(
             // Override with official assignments
             for (const assignment of assignmentsByApplicantId[applicant.id] ||
                 []) {
+                let hours = assignment.hours || 0;
```

Comment:
Something's making me uncomfortable about the order of code here.

I think it would be better to have an `activeOfferStatusToStatus(...)` function. Then you can check `hours = status === "unassignable" ? 0 : assignment.hours || 0`